.gitmodules

```diff
@@ -1,9 +0,0 @@
-[submodule "pybind11"]
-	path = pybind11
-	url = https://github.com/pybind/pybind11.git
-[submodule "googletest"]
-	path = googletest
-	url = https://github.com/google/googletest.git
-[submodule "tests/toml-test"]
```

Comment:
Why remove `toml-test`? The python unit tests need this module to work.

Reply:
Oops sorry will add that back.